extra/3RD_PARTY_QUICK_START.md

```diff
@@ -59,12 +66,11 @@ Publish it to NPM also if you'd like. This will make it much easier for anyone u
 
 When your language module is ready create a PR that adds it to the `README.md` file in the list of languages.
 
-
 ## The Future
 
 More work could be done on:
 
 - Allowing you to ONLY run your own tests, not the whole suite.
-- Allowing you to maintain a 3rd party module WITHOUT it being inside of a `highlight-js` checkout (this requires discussion though)
+- ~~Allowing you to maintain a 3rd party module WITHOUT it being inside of a `highlight-js` checkout (this requires discussion though)~~
```

Comment:
Did you cross this out because your feel the discussion is over, or?

Reply:
related to your comment above, my initial approach was start with a highlightjs-xyz repo firs. but I agree, the better process is start with your own repo first, and transfer to highlightjs after approval. I'll update the docs to promote this process instead of what I had.